index.js

```diff
@@ -66,28 +67,59 @@ exports.init = function (ssb, config) {
     )
   }
 
+  function monitorGroupMembersStream() {
+    if (!ssb.tribes2) return
+
+    membersInterval = setInterval(() => {
```

Comment:
ugh, it would be really nice with a pull based api for list.

Reply:
Yes, this setInterval is nasty. I already made tasks in Airtable to update ssb-box2, and ssb-tribes2 to have a proper live stream, and Jacob is assigned to those.